src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -1971,6 +2019,963 @@ enum Nf {
 
 #undef INSN
 
+// ========================================
+// RISC-V Compressed Instructions Extension
+// ========================================
+// Note:
+// 1. When UseRVC is enabled, 32-bit instructions under 'CompressibleRegion's will be
+//    transformed to 16-bit instructions if compressible.
+// 2. RVC instructions in Assembler always begin with 'c_' prefix, as 'c_li',
+//    but most of time we have no need to explicitly use these instructions.
+// 3. We introduce 'CompressibleRegion' to hint instructions in this Region's RTTI range
```

Comment:
Suggestion:

// 3. 'CompressibleRegion' is introduced to hint instructions in this Region's RTTI range